WebRoot/js/zimbraMail/prefs/view/ZmFilterPage.js

```diff
@@ -36,6 +36,7 @@
  */
 ZmFilterPage = function(parent, section, controller) {
 	ZmPreferencesPage.apply(this, arguments);
+	this.addClassName("ZmFilterPage");
```

Comment:
Well, parent constructor for now doesn't seems to support className param for now. (https://github.com/Zimbra/zm-web-client/blob/develop/WebRoot/js/zimbraMail/prefs/view/ZmPreferencesPage.js),  we will need to update `ZmPreferencesPage` class to have support for `className` param.

Do you feel we should go ahead and have this changes in `ZmPreferencesPage`?

Let me know, thanks.